.github/workflows/examples-linux-standalone.yaml

```diff
@@ -128,6 +128,14 @@ jobs:
                     out/ota_requestor_debug/chip-ota-requestor-app \
                     /tmp/bloat_reports/
 
+            - name: Build example Standalone Door Lock App
+              timeout-minutes: 5
+              run: |
+                  scripts/examples/gn_build_example.sh examples/lock-app/linux out/lock_app_debug
```

Comment:
could you add it (and use) build_examples.py instead?

Reply:
Done